.github/workflows/ci.yml

```
@@ -34,9 +34,28 @@ jobs:
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+          cache: 'pnpm'
+
+      - run: pnpm install --frozen-lockfile
+
+      - name: Lint extension translation keys
+        run: pnpm --filter @ancore/extension-wallet lint:translations
+
   package:
```

actionlint (1.7.12)

[error] "runs-on" section is missing in job "package"

(syntax-check)